src/osgPlugins/OpenFlight/Record.cpp

```diff
@@ -84,6 +84,14 @@ void flt::insertMatrixTransform(osg::Node& node, const osg::Matrix& matrix, int
     osg::ref_ptr<osg::Node> ref = &node;
     osg::Node::ParentList parents = node.getParents();
 
+    // Disconnect node from parents.
```

Comment:
I am not seeing any changes to this file in makOsg 3.4 so we probably want the 3.6 version of this file.

Reply:
Replaced file with OSG 3.6 version.